perl/lib/Sanger/CGP/Star/Implement.pm

```diff
@@ -598,6 +599,23 @@ sub _which {
 	return $path;
 }
 
+sub _to_commandline_safe_tag_for_star {
+	# according to this: https://unix.stackexchange.com/a/398649
+	# preserving tag values is complicated
+	my $tag = shift;
+	# first skip back slashes
+	$tag =~ s/\\/\\\\/g;
+	# then skip other needed-to-skip characters.
+	# this will inevitably introduce an extra back slash into the tag if there's an '!', but for now no other way to skip '!' to prevent bash history expansion.
+	my @need_to_skips = ('$', '`', '"', '!');
```

Comment:
Also, this would be more readable:

```
my @need_to_skips = (q{$}, q{`}, q{"}, q{!});
```

See: https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitNoisyQuotes